### PR TITLE
Initialize plenum induced air node conditions fix 4926

### PR DESCRIPTION
**Pull request by mjwitte**
Addresses #4926 The node conditions for Induced Air Outlet Nodes in AirLoopHVAC:ReturnPlenum were being set to zero (temperature, humidity ratio, etc.).  Induced air nodes may be used by AirTerminal:SingleDuct:SeriesPIU:Reheat (and ParallelPIU) as a Secondary Air Inlet Node.  This fix sets the induced air node conditions to match the return plenum zone node conditions at the end of the previous time step (same as any outlet from a plenum zone).

### CODE REVIEW

**Comment by mjwitte** [comment]
@wfbuhl Here is a proposed fix.  Please review.

**Comment by mjwitte** [comment]
This does not address CO2 and generic contaminant concentrations yet.  That may require more thought.

**Comment by mjwitte** [comment]
@lgu1234 Thanks for reviewing this.  I will implement your suggestion.

**Comment by Myoldmopar** [comment]
@mjwitte pull develop in one more time to get the regressions fully running...

**Comment by mjwitte** [comment]
@Myoldmopar Strange regression failure on Linux for regression.SupermarketSubCoolersVariableSuction

**Comment by Myoldmopar** [comment]
Just saw that too, unrelated to the actual regression errors, it had trouble connecting with Amazon to upload the regression files.  I'll delete those results and see if it works.

File: src/EnergyPlus/ZonePlenum.cc [hunk 1]
```diff
@@ -1138,22 +1161,19 @@ namespace ZonePlenum {
 				}
 				Node( ZoneNode ).CO2 = Node( OutletNode ).CO2;
 			} else {
-				Node( OutletNode ).CO2 = Node( InletNode ).CO2;
-				Node( ZoneNode ).CO2 = Node( InletNode ).CO2;
+				Node( OutletNode ).CO2 = Node( ZoneNode ).CO2;
```
Comment (Myoldmopar) [inline src/EnergyPlus/ZonePlenum.cc:1164]: This is the functional change to fix this PR, right?  It previously just moved the inlet node info to the outlet and zone nodes.  Now it moves the zone node to the outlet node and that is all?  So where the is ZoneNode updated now?  I see calls above to set the InducedNode conditions; is that it?

File: src/EnergyPlus/ZonePlenum.cc [hunk 2]
```diff
@@ -718,17 +723,28 @@ namespace ZonePlenum {
...
-		// Set the induced air flow rates
+		ZoneNodeNum = ZoneRetPlenCond( ZonePlenumNum ).ZoneNodeNum;
+		// Set the induced air flow rates and conditions
 		for ( NodeNum = 1; NodeNum <= ZoneRetPlenCond( ZonePlenumNum ).NumInducedNodes; ++NodeNum ) {
 			InducedNode = ZoneRetPlenCond( ZonePlenumNum ).InducedNode( NodeNum );
 			ZoneRetPlenCond( ZonePlenumNum ).InducedMassFlowRate( NodeNum ) = Node( InducedNode ).MassFlowRate;
 			ZoneRetPlenCond( ZonePlenumNum ).InducedMassFlowRateMaxAvail( NodeNum ) = Node( InducedNode ).MassFlowRateMaxAvail;
 			ZoneRetPlenCond( ZonePlenumNum ).InducedMassFlowRateMinAvail( NodeNum ) = Node( InducedNode ).MassFlowRateMinAvail;
+
+			ZoneRetPlenCond( ZonePlenumNum ).InducedTemp( NodeNum ) = Node( ZoneNodeNum ).Temp;
```
Comment (mjwitte) [inline src/EnergyPlus/ZonePlenum.cc:734]: @Myoldmopar The main fix is here - these weren't being set at all.

File: src/EnergyPlus/ZonePlenum.cc (hunk 1 above)
Comment (mjwitte) [inline src/EnergyPlus/ZonePlenum.cc:1164]: @Myoldmopar This change was a little more subtle.  There is code in ZoneContaminantPredictorCorrector to calculate the CO2 and GenContam concentrations for the plenum zone.  Originally I proposed doing only the else block, but @lgu1234  suggested that the plenum object should update the CO2 and GenContam concentrations here when there is flow, as well as in the ZoneContaminantPredictorCorrector.

**Comment by Myoldmopar** [comment]
Thanks for responding @mjwitte, this seems to pass all the CI and my mental checklist items.  I'll merge unless you have additional work to do from other people's review.

**Comment by mjwitte** [comment]
@Myoldmopar That's all I'm aware of for now.

**Pull request merged**